src/i18n/locales/en.ts

```diff
@@ -618,6 +618,24 @@ export const en = {
   application: 'Application',
   multiview: 'Multiview',
   setting_up: 'Setting up connections to external API:s...',
+  teardown: {
+    name: 'Teardown',
+    warning: 'WARNING!',
+    tearing_down: 'Tearing down...',
+    results: 'Teardown results',
+    are_you_sure: 'Are you sure?',
+    description: 'You are about to:',
+    optional: 'Optional:',
+    reset_pipelines: 'Reset all pipelines',
+    delete_ingest_srt_sources: 'Delete all ingest SRT sources',
+    pipeline_output_streams: 'Delete all pipeline output streams',
+    pipeline_multiviewers: 'Delete all pipeline multiviewer outputs',
+    pipeline_streams: 'Delete all pipeline streams',
+    pipeline_control_connections: 'Delete all pipeline control connections',
+    ingest_streams: 'Delete all ingest streams',
+    ingest_src_sources: 'Delete all ingest SRT sources',
```

Comment:
necessary to have two identical? 'Delete all ingest SRT sources' appears twice, both here and in the sv.ts

Reply:
I don't see what you mean.

Reply:
```
delete_ingest_srt_sources: 'Delete all ingest SRT sources',
    pipeline_output_streams: 'Delete all pipeline output streams',
    pipeline_multiviewers: 'Delete all pipeline multiviewer outputs',
    pipeline_streams: 'Delete all pipeline streams',
    pipeline_control_connections: 'Delete all pipeline control connections',
    ingest_streams: 'Delete all ingest streams',
    ingest_src_sources: 'Delete all ingest SRT sources',
```

delete_ingest_srt_sources and ingest_src_sources have the same text